README.md

```diff
@@ -214,7 +215,47 @@ Any files uploaded by a user will never be sent directly to the frontend service
 
 # Local development
 
-## Setup
+## Developing services that use the CDP-Uploader
+
+If your service is going to use the CDP-Uploader to receive files you may want to start by running the uploader locally. The easiest way to do this is using `docker compose`.
+
+#### Docker Compose
+
+The CDP-Uploader project provide as base [compose.yml](compose.yml) file to get you started.
+
+Copy [compose.yml](compose.yml) as well as the [./compose](./compose) folder into your own project and running `docker compose pull` and then `docker compose up`.
+
+This will start:
+
+- redis
+- localstack (a local AWS emulator)
+- cdp-uploader
+
+It will also inject a [start-up script](./compose/start-localstack.sh) into the localstack container that automatically creates the queues and buckets needed by the uploader as well as a test bucket named 'my-bucket'.
+If your service requires its own bucket you can add a line to [start-localstack.sh](./compose/start-localstack.sh) script to create one.
+
+```yaml
+aws --endpoint-url=http://localhost:4566 s3 mb s3://cdp-uploader-quarantine
+aws --endpoint-url=http://localhost:4566 s3 mb s3://my-bucket
+
+## Insert your bucket here, e.g.
```

Comment:
Is this meant to be a title?

Reply:
no, its a comment in the example shell file